velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -913,5 +913,69 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(0, "yyyy-MM-dd HH:II"), "Specifier I is not supported");
 }
 
+TEST_F(DateTimeFunctionsTest, makeYMInterval) {
+  const auto getYMInterval = [](const std::optional<int32_t>& ymInterval) {
```

Comment:
This shouldn't be needed.

Use `INTERVAL_YEAR_MONTH()->valueToString(*interval)` in the caller directly.